carbonserver/carbonserver/api/infra/repositories/repository_organizations.py

```diff
@@ -115,16 +114,6 @@ def get_organization_detailed_sums(
                         "emissions_count"
                     ),
                 )
-                .join(
-                    SqlModelTeam,
-                    SqlModelOrganization.id == SqlModelTeam.organization_id,
```

Comment:
This needs a .join between the `SqlModelOrganization.id == SqlModelProject.organization_id`. But it does not exist yet this relationship Project--> Organization. @prmths128 What do you think of first merging #577 (it will add the relationship) and then merge this PR?

Reply:
merged